app/vlselect/main.go

```diff
@@ -321,4 +392,13 @@ var (
 
 	// no need to track duration for tail requests, as they usually take long time
 	logsqlTailRequests = metrics.NewCounter(`vl_http_requests_total{path="/select/logsql/tail"}`)
+
+	rulesRequests = metrics.NewCounter(`vl_http_requests_total{path="/select/api/v1/rules"}`)
+	rulesDuration = metrics.NewSummary(`vl_http_requests_duration_seconds{path="/select/api/v1/rules"}`)
```

Comment:
A small question: do we need `Summary` for vmalert requests? Since they are typically fast and unrelated to vlogs logic, and might negatively impact the vl_http_requests_duration_seconds p99?

Reply:
just added it as for other metrics, but vmalert may have this metrics, as well, so will remove it